src/app/signup/page.tsx

```diff
@@ -8,7 +8,59 @@ import { useForm } from "react-hook-form";
 import { signupUser } from "@/lib/User/actions";
 import { signUpSchema } from "@/lib/User/validations";
 
+import { useState } from "react";
 const SignUp = () => {
+  const [showPassword, setShowPassword] = useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
+  const [password, setPassword] = useState("");
```

Comment:
We shouldn't need to store the `password` or `confirmPassword` state. That is being stored via `react-hook-form`